db/db_with_timestamp_basic_test.cc

```diff
@@ -2784,6 +2784,196 @@ TEST_F(DBBasicTestWithTimestamp, MultiGetNoReturnTs) {
   Close();
 }
 
+TEST_F(DBBasicTestWithTimestamp, SingleDeleteForwardIterateStartSeqnum) {
```

Comment:
This looks similar to `ForwardIterateStartSeqnum`. Can we combine them by defining another parameterized gtest class, e.g. `DBBasicDeletionTestWithTimestamp` or something like that?